src/_actions/feedback.action.js

```diff
@@ -0,0 +1,23 @@
+import constants from '../_constants';
+import axios from 'axios';
+
+const { LOAD_FEEDBACK_SUCCESS, LOAD_FEEDBACK_FAILURE } = constants;
+
+export const feedbackAction = (page, limit = 10) => {
+    return (dispatch) => {
+      axios.get(`feedback?_page=${page}&_limit=${limit}`)
+      .then((response) => {
+        return dispatch({
+          type: LOAD_FEEDBACK_SUCCESS,
+          payload: response,
+        });
+      })
+      .catch((error) => {
+        return dispatch({
+          type: LOAD_FEEDBACK_FAILURE,
+          payload: error,
+        });
+      });
+    }
+  };
+  
```

Comment:
Add new line. It will be easier for you if you set this in your editor config so as not to worry about it each time.

For example, I'm using VS Code and I have the following in my settings:
```
{
  "files.trimTrailingWhitespace": true,
  "files.insertFinalNewline": true,
  "files.trimFinalNewlines": true
}
```
I also added the following to the editor settings to ensure javascript and html files will always have an indentation of 2. Note that if there is already an existing file with an indentation of 4, then add ` "editor.detectIndentation": false` for the file's language to stop using auto-detection and for the set tab size to work
```
{
  "[javascript]": {
    "editor.tabSize": 2
  },
  "[html]" : {
    "editor.tabSize: 2
  }
}
```

Reply:
hey @sachungo , thank you for this!